src/index.html

```diff
@@ -12,7 +15,132 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <section class="calendar calendar--start-day-sun calendar--month-length-31">
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
+      <a
+        href="#"
+        class="calendar__day"
+      ></a>
```

Comment:
It seems like you're manually creating each day of the calendar. This is not a scalable or maintainable solution. What if the month has more or fewer days? Instead, consider creating these elements dynamically using JavaScript. This way, you can easily adjust the number of days based on the month.